### PR TITLE
PR #train and model documentation

### PR DESCRIPTION
**Pull request by sblackburn86**
Documented missing functions in models.py and train.py as in issue #29

### CODE REVIEW

File: ecodse_funtime_alpha/models.py [hunk 1]
```diff
@@ -4,7 +4,32 @@
...
+    Methods
+    -------
+    call:
+        inherit from keras
+        Usage example: model(x)
+    """
+
     def __init__(self, hiddensize, outsize):
+        """
+        Class constructure
```
Comment (hbertrand) [inline ecodse_funtime_alpha/models.py]: Typo

File: ecodse_funtime_alpha/train.py [hunk 2]
```diff
@@ -21,6 +41,28 @@ def batch_dataset(dataset, nepoch, batchsize):
...
+    Gradient is applied manually on the model using the optimizer.
+
+    Parameters
+    ----------
+    dataset : tf dataset
+        original dataset (unshuffled, not-split into mini-batches)
+    model : tf.keras.Model
+        an initialized model working in eager execution mode
+    optimizer : tf.train.Optimizer
+        tensorflow optimizer (e.g. tf.train.AdamOptimizer()) to train the model
```
Comment (hbertrand) [inline ecodse_funtime_alpha/train.py]: Code should be in between `` so it looks nice when building the doc. Here it should be`tf.train.AdamOptimizer()`

File: ecodse_funtime_alpha/train.py [hunk 3]
```diff
@@ -43,6 +106,19 @@ def fit_loop(dataset, model, optimizer, nepoch, batchsize):
...
+    read and parse the arguments
+
+    Parameters
+    ----------
+    args : sys.argv
+        arguments specified by the user
+
+    Returns
+    -------
+    ArgumenterParser object
```
Comment (hbertrand) [inline ecodse_funtime_alpha/train.py]: Typo

File: ecodse_funtime_alpha/train.py (hunk 2 above)
Comment (sblackburn86) [reply ecodse_funtime_alpha/train.py]: solved. At line 87 as well.

File: ecodse_funtime_alpha/train.py (hunk 3 above)
Comment (sblackburn86) [reply ecodse_funtime_alpha/train.py]: solved

**Pull request merged**